src/allmain.c

```diff
@@ -6,6 +6,7 @@
 /* various code that was replicated in *main.c */
 
 #include "hack.h"
+#include "nle.h"
```

Comment:
Let's not include nle.h here.

Instead, let's move the `extern int nle_spawn_monsters;` line from nle.h into this file (with a comment).